### DISCUSSION TITLE
Cannot remove .grid from Tailwind output.

### DISCUSSION THREAD

**Discussion by ronnross** [tailwindlabs/tailwindcss#7274] [Help] [answered]
We want to add tailwind to our site and transition over. Problem is we have a .grid already defined using flexbox. Eventually this will change, but in the time being it would be nice to remove .grid from tailwinds. I have searched the docs and older issue, but no luck. I have disabled all I can 
`corePlugins: {
    gap: false,
    columnGap: false,
    rowGap: false,
    gridColumn: false,
    gridColumnEnd: false,
    gridColumnStart: false,
    gridRow: false,
    gridRowEnd: false,
    gridRowStart: false,
    gridTemplateColumns: false,
    gridTemplateRows: false,
    gridAutoFlow: false,
  },`

But no matter what I do `.grid` still is in the output file. Can someone help?

**Comment by diahron** [comment]
Here is a website that should help you transition over. "Tailwind Labs". This is a Youtube channel that will step you through the process of transitioning your site to Tailwind. The site is awesome. His tutorials are very precise and clear.

**Comment by ExEr7um** [comment, marked as answer]
I think Tailwind generates it in `display` core plugin, so you should try this:
```javascript
corePlugins: {
  display: false,
}
```
